repo/packages/O/openldap/0/marathon.json.mustache

```diff
@@ -0,0 +1,63 @@
+{
+    "id": "{{service.name}}",
+    "cpus": {{openldap.cpus}},
+    "mem": {{openldap.mem}},
+    "instances": 1,
+    "env": {
+        "LDAP_ORGANISATION": "{{openldap.organisation}}",
+        "LDAP_DOMAIN": "{{openldap.domain}}",
+        "LDAP_ADMIN_PASSWORD": "{{openldap.admin_password}}",
+        "LDAP_READONLY_USER": "{{openldap.readonly_user}}",
+        "LDAP_READONLY_USER_USERNAME": "{{openldap.readonly_username}}",
+        "LDAP_READONLY_USER_PASSWORD": "{{openldap.readonly_password}}",
+        "LDAP_TLS_VERIFY_CLIENT": "{{openldap.tls_verify_client}}"
+    },
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "{{resource.assets.container.docker.openldap-docker}}",
+            "forcePullImage": true,
+            "network": "BRIDGE",
+            "portMappings": [
+            {
+                "containerPort": 389,
+                "hostPort": 0,
+                "protocol": "tcp"
+            },
+            {
+                "containerPort": 636,
+                "hostPort": 0,
+                "protocol": "tcp"
+            }
+            ]
+        },
+        "volumes": [
+        {
+            "containerPath": "/var/lib/ldap",
+            "hostPath": "{{openldap.host_volume}}/{{service.name}}/ldap",
+            "mode": "RW"
+        },
+        {
+            "containerPath": "/etc/ldap/slapd.d",
+            "hostPath": "{{openldap.host_volume}}/{{service.name}}/slapd.d",
+            "mode": "RW"
+        }
+        ]
+    },
+    "acceptedResourceRoles": ["slave_public"],
```

Comment:
Will fix!

On Mon, Apr 18, 2016 at 4:01 PM, Thomas Rampelberg <notifications@github.com
> wrote:

> In repo/packages/O/openldap/0/marathon.json.mustache
> <https://github.com/mesosphere/universe/pull/415#discussion_r60149137>:
>
> > +            ]
> > +        },
> > +        "volumes": [
> > +        {
> > +            "containerPath": "/var/lib/ldap",
> > +            "hostPath": "{{openldap.host_volume}}/{{service.name}}/ldap",
> > +            "mode": "RW"
> > +        },
> > +        {
> > +            "containerPath": "/etc/ldap/slapd.d",
> > +            "hostPath": "{{openldap.host_volume}}/{{service.name}}/slapd.d",
> > +            "mode": "RW"
> > +        }
> > +        ]
> > +    },
> > +    "acceptedResourceRoles": ["slave_public"],
>
> I'm reasonably sure I'd never want to run my LDAP server on a public
> instance.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/universe/pull/415/files/ff15193a05bbbed182306b9f236c9b499f43656c#r60149137>
>